pygmt/tests/test_logo.py

```diff
@@ -1,34 +1,26 @@
 """
 Tests for fig.logo.
 """
+import pytest
 from pygmt import Figure
-from pygmt.helpers.testing import check_figures_equal
 
 
-@check_figures_equal()
+@pytest.mark.mpl_image_compare
 def test_logo():
     """
     Plot a GMT logo of a 2 inch width as a stand-alone plot.
     """
-    fig_ref, fig_test = Figure(), Figure()
-    # Use single-character arguments for the reference image
-    fig_ref.logo(D="x0/0+w2i")
-    fig_test.logo(position="x0/0+w2i")
-    return fig_ref, fig_test
+    fig = Figure()
+    fig.logo(position="x0/0+w2i")
+    return fig
 
 
-@check_figures_equal()
+@pytest.mark.mpl_image_compare
 def test_logo_on_a_map():
     """
     Plot a GMT logo in the upper right corner of a map.
     """
-    fig_ref, fig_test = Figure(), Figure()
-    # Use single-character arguments for the reference image
-    fig_ref.coast(R="-90/-70/0/20", J="M6i", G="chocolate", B="")
-    fig_ref.logo(D="jTR+o0.1i/0.1i+w3i", F="")
-
-    fig_test.coast(
-        region=[-90, -70, 0, 20], projection="M6i", land="chocolate", frame=True
-    )
-    fig_test.logo(position="jTR+o0.1i/0.1i+w3i", box=True)
-    return fig_ref, fig_test
+    fig = Figure()
+    fig.coast(region=[-90, -70, 0, 20], projection="M6i", land="chocolate", frame=True)
+    fig.logo(position="jTR+o0.1i/0.1i+w3i", box=True)
```

Comment:
Perhaps rewrite the test to:

- Use SI units, not US units
- Use smaller region in Figure.coast() and lower resolution coastline data, to reduce the baseline image size (although it doesn't matter much now).

Reply:
I think we can remove the `fig.coast` call entirely to make it a proper unit test. I.e. just do `fig.logo` with position and box arguments.

Reply:
Yes, that would be better.

Reply:
Ok, done at af79eeff69592c25fb406c1513eb6da427b11434